data/sql/updates/pending_db_world/rev_1697941424465768700.sql

```diff
@@ -8,6 +8,20 @@ INSERT INTO `spell_custom_attr` (`spell_id`, `attributes`) VALUES (28282, 671088
 UPDATE `gameobject_template_addon` SET `flags` = 34
 WHERE (`entry` = 104591);
 
+
+-- Add a trigger for Ashbringer events Fire once after approaching
+DELETE
+FROM `areatrigger_scripts`
+WHERE `entry` = 4089;
+INSERT INTO `areatrigger_scripts` (`entry`, `ScriptName`) VALUES
+ (4089, 'at_scarlet_monastery_cathedral_entrance');
+DELETE
+FROM `areatrigger`
+WHERE `entry`=4089;
+INSERT INTO `areatrigger` (`entry`, `map`, `x`, `y`, `z`, `radius`, `length`, `width`, `height`, `orientation`) VALUES
+ (4089, 189, 864.623, 1321.6, 18.0683, 5, 0, 0, 0, 0);
```

Comment:
This areatrigger already exists in the world DB on latest revision and does not need to be added here.